.github/workflows/build-runner-images.yml

```
@@ -52,7 +52,7 @@ jobs:
           echo "PKR_VAR_subnet_id=$SUBNET_ID" >> "$GITHUB_ENV"
 
       - name: Setup `packer`
-        uses: hashicorp/setup-packer@main
+        uses: hashicorp/setup-packer@v2.0.1
```

I'm generally a fan of pinning to a specific version, but for the rest of these actions we've just been sticking to major, and I think that will work here for now. We can get more specific when it breaks on us again 😁 
```suggestion
        uses: hashicorp/setup-packer@v2
```

---

Thanks for checking that. Issue filed: https://github.com/hashicorp/setup-packer/issues/93